compiler/modules/types/core.bal

```diff
@@ -22,7 +22,7 @@ public type UniformTypeCode
     |UT_XML_RW|UT_LIST_RW|UT_MAPPING_RW|UT_TABLE_RW|UT_OBJECT_RW
     |UT_STREAM|UT_FUTURE;
 
-type Atom RecAtom|TypeAtom;
+public type Atom RecAtom|TypeAtom;
```

Comment:
I don't want this change to expose internal implementation details of this module that were not previously exposed.

Reply:
fixed